src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -335,4 +335,12 @@ private Request matchLoansToRequest(
       .withLoan(loanMap.getOrDefault(request.getItemId(), null));
   }
 
+  public CompletableFuture<Result<MultipleRecords<Loan>>> findOpenLoansByUserId(
+    LoanAndRelatedRecords loanAndRelatedRecords) {
+    String userId = loanAndRelatedRecords.getLoan().getUser().getId();
+    final Result<CqlQuery> statusQuery = getStatusCQLQuery("Open");
+    final Result<CqlQuery> userIdQuery = exactMatch("userId", userId);
```

Comment:
Add loanPolicyId to this query to look for user loans of particular type, not all Open user loans

Reply:
done

Reply:
Why is it important to find loans using the loan policy ID? As I understand it, that does not meet the expected requirements as it might filter out loans that should be included in the count yet have a different loan policy.